src/helpers/middleware.js

```diff
@@ -1,57 +1,109 @@
 import { check, validationResult } from 'express-validator/check';
 import model from '../model/meetUpModel';
 
+const validatorFunction = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({
+      status: 400,
+      errors: errors.array(),
+    });
+  }
+  return next();
+};
+
 const middleware = {
   meetUp: [
-    check('id').escape().isLength({ min: 1, max: 4 }).withMessage('must be minimum of 1-4 letter').isInt().withMessage('must be an integer'),
-    check('topic').escape().isLength({ min: 10, max: 30 }).withMessage('must be minimum of 10-30 letters').isString(),
-    check('location').escape().isLength({ min: 5, max: 10 }).withMessage('must be minimum of 5-10 letters').isString(),
-    check('happeningOn').escape().isLength({ min: 5, max: 10 }).withMessage('must be minimum of 5-10 letters'),
-    check('createdOn').escape().isLength({ min: 5, max: 10 }).withMessage('must be minimum of 5-10 letters'),
-    (req, res, next) => {
-      const errors = validationResult(req);
-      if (!errors.isEmpty()) {
-        return res.status(400).json({
-          status: 400,
-          errors: errors.array(),
-        });
-      }
-      return next();
-    },
+    check('id')
```

Comment:
Good work on raising this PR however your code should be DRY. Consider refactoring. Too much repetition here.

Reply:
Alright